tools/codegen/codegen.py

```diff
@@ -33,10 +33,14 @@
 flags.DEFINE_multi_string("load", [], "List of data files to load.")
 flags.DEFINE_multi_string("override", [], "List of key=value pair data.")
 flags.DEFINE_string("schema", None, "JSON schema to check against.")
-flags.DEFINE_multi_string("output", None, "Output files to generate.")
```

Comment:
We need FLAGS.output to be a multi_string for use in multigen mode.

Reply:
Not really, in the original code, when `--multigen_mode` was activated it was only accepted one output file: https://github.com/enfabrica/enkit/blob/master/tools/codegen/codegen.py#L237

Reply:
Thanks for the clarifications on the call. I changed the implementation to only block calls where multiple templates are informed and output is specified.